src/components/forms/Selects.tsx

```diff
@@ -23,7 +23,6 @@ export const BoatType = ({ boatTypes, required, className }: BoatTypeProps) => (
     label="form.registered.field.type.label"
     required={required}
   >
-    <option>-</option>
```

Comment:
will this completely remove this option from the dropdown here?
![image](https://user-images.githubusercontent.com/24206160/88769941-c12c8f80-d185-11ea-8a8e-2b9dde18b519.png)
will the first `BoatType` be chosen then automatically?

Reply:
Yes that's the idea, only valid boat types can then be selected. Having the extra option `-` selectable messes with the `required` validation. Though we could perhaps make `-` a special case in the required validator for `Select`.

Reply:
hm, not sure if it will result in users forgetting to choose the right type for their boat 🤔  ideally, some empty `option` is the default, but submitting the form without choosing some actual `BoatType` is now allowed.

Reply:
Changed the validation logic for all `Select` components.

Reply:
nice! that should do the trick 👍